backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -5,13 +5,18 @@
 import gov.cdc.usds.simplereport.api.model.errors.IllegalGraphqlArgumentException;
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
+import java.util.Date;
+
 
 /**
  * Static package for utilities to translate things to or from wireline format in non copy-paste ways.
  */
 public class Translators {
 
     private static DateTimeFormatter US_SLASHDATE_FORMATTER = DateTimeFormatter.ofPattern("MM/dd/yyyy");
+	private static DateTimeFormatter ISO_INSTANT_FORMATTER = DateTimeFormatter.ISO_INSTANT;
```

Comment:
But then we don't seem to use it? If using this and using `Instant.parse` are equivalent, I'd prefer to use this.